tensorflow/training/docker/2.10/py3/cu112/Dockerfile.gpu

```diff
@@ -310,100 +310,103 @@ CMD ["/bin/bash"]
-#ARG SMDEBUG_TAG=1.0.17
-#RUN git clone https://github.com/awslabs/sagemaker-debugger.git \
-#    && cd sagemaker-debugger \
-#    && git checkout tags/$SMDEBUG_TAG -b $SMDEBUG_TAG \
-#    && $PYTHON setup.py install && cd .. && rm -rf sagemaker-debugger
-#
-## install sagemaker-tensorflow from source
-#ARG SAGEMAKER_TENSORFLOW_TAG=v1.15.0
-#RUN apt update && apt install -y libcurl4-openssl-dev
-#RUN git clone -b tf-2 https://github.com/aws/sagemaker-tensorflow-extensions.git \
-#    && cd sagemaker-tensorflow-extensions \
-#    && git checkout tags/$SAGEMAKER_TENSORFLOW_TAG -b $SAGEMAKER_TENSORFLOW_TAG \
-#    && $PYTHON -m pip install . \
-#    && cd .. && rm -rf sagemaker-tensorflow-extensions
-#
-## install boost
-## tensorflow is compiled with --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=1"
-#RUN wget https://sourceforge.net/projects/boost/files/boost/1.73.0/boost_1_73_0.tar.gz/download -O boost_1_73_0.tar.gz \
-#   && tar -xzf boost_1_73_0.tar.gz \
-#   && cd boost_1_73_0 \
-#   && ./bootstrap.sh \
-#   && ./b2 define=_GLIBCXX_USE_CXX11_ABI=1 threading=multi --prefix=/usr -j 64 cxxflags=-fPIC cflags=-fPIC install || true \
-#   && cd .. \
-#   && rm -rf boost_1_73_0.tar.gz \
-#   && rm -rf boost_1_73_0 \
-#   && cd /usr/include/boost
-#
-## add smdataparallel
-## smdataparallel
-#ARG SMDATAPARALLEL_BINARY=https://smdataparallel.s3.amazonaws.com/binary/tensorflow/2.9.0/cu112/2022-05-11/smdistributed_dataparallel-1.4.1-cp39-cp39-linux_x86_64.whl
-#
-## Install SMD DP binary
-#RUN SMDATAPARALLEL_TF=1 ${PYTHON} -m pip install --no-cache-dir ${SMDATAPARALLEL_BINARY}
-#
-## Add NGC vars
-#ENV TF_AUTOTUNE_THRESHOLD=2
-#
-#ENV LD_LIBRARY_PATH="/usr/local/lib/${PYTHON}/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"
-#
-## remove tmp files
-#RUN rm -rf /tmp/*
-#
-#RUN HOME_DIR=/root \
-#   && curl -o ${HOME_DIR}/oss_compliance.zip https://aws-dlinfra-utilities.s3.amazonaws.com/oss_compliance.zip \
-#   && unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ \
-#   && cp ${HOME_DIR}/oss_compliance/test/testOSSCompliance /usr/local/bin/testOSSCompliance \
-#   && chmod +x /usr/local/bin/testOSSCompliance \
-#   && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
-#   && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
-#   && rm -rf ${HOME_DIR}/oss_compliance*
-#   
-#CMD ["/bin/bash"]
+FROM common AS sagemaker
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG TF_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/tensorflow/r2.10_aws/gpu/2022-09-07-02-07/tensorflow_gpu-2.10.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+
+# sagemaker-specific environment variable
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+# https://github.com/tensorflow/models/issues/9267
+# tf-models does not respect existing installations of TF and always installs open source TF
+RUN ${PIP} install --no-cache-dir -U \
+    tf-models-official==2.9.2
```

Comment:
2.10.0 has been released.

Reply:
Good catch, thanks @Lokiiiiii.  I added a commit to bump the version for `tf-models-official` to `2.10.0`.